frontend/components/MainPage/header/LastTransaction.tsx

```diff
@@ -0,0 +1,79 @@
+import { Skeleton, Typography } from 'antd';
+import { useCallback, useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { useInterval } from 'usehooks-ts';
+
+import { useAddress } from '@/hooks/useAddress';
+import { getLatestTransaction } from '@/service/Ethers';
+import { TransactionInfo } from '@/types/TransactionInfo';
+import { getTimeAgo } from '@/utils/time';
+
+const { Text } = Typography;
+
+const Loader = styled(Skeleton.Input)`
+  line-height: 1;
+  span {
+    width: 120px !important;
+    height: 12px !important;
+    margin-top: 6px !important;
+  }
+`;
+
+const POLLING_INTERVAL = 60 * 1000; // 1 minute
+
+/**
+ * TODO:
+ * - add cache if the same block number is requested
+ *
+ * Components to display the last transaction time and link to the transaction on GnosisScan
+ * by agent safe.
+ */
+export const LastTransaction = () => {
+  const { multisigAddress } = useAddress();
+
+  const [isFetching, setIsFetching] = useState(true);
+  const [transaction, setTransaction] = useState<TransactionInfo | null>(null);
+
+  const fetchTransaction = useCallback(async () => {
+    if (!multisigAddress) return;
+
+    getLatestTransaction(`${process.env.GNOSIS_RPC}`, multisigAddress)
```

Comment:
does it make sense to pass the rpc into the getLatestTransaction? if we use gnosis anyway, can we just use `process.env.GNOSIS_RPC` inside the func?